src/deprecated/event.js

```diff
@@ -28,7 +28,8 @@ jQuery.fn.extend( {
 	}
 } );
 
-jQuery.each( ( "blur focus focusin focusout resize scroll click dblclick " +
+jQuery.each(
+	( "blur focus focusin focusout resize scroll click dblclick " +
```

Comment:
Was this change necessary?

Reply:
The current code needs to be modified as follows according to the new rules, and changes of 9 lines occur.

```
jQuery.each( ( "blur focus focusin focusout resize scroll click dblclick " +
	"mousedown mouseup mousemove mouseover mouseout mouseenter mouseleave " +
	"change select submit keydown keypress keyup contextmenu" ).split( " " ),
function( _i, name ) {   // <-- start

	// Handle event binding
	jQuery.fn[ name ] = function( data, fn ) {
		return arguments.length > 0 ?
			this.on( name, null, data, fn ) :
			this.trigger( name );
	};
} );                   // <-- end
```

Reply:
If there is a better way, please let me know.

Reply:
Thanks, the current way makes sense to me then.